docs/release-notes/8.14.asciidoc

```diff
@@ -0,0 +1,56 @@
+[[release-notes-header-8.14.0]]
+== 8.14
+
+[discrete]
+[[release-notes-8.14.0]]
+=== 8.14.0
+
+[discrete]
+[[features-8.14.0]]
+==== New features
+* Introduces Attack discovery, a new feature that uses AI to identify potential attacks and help you quickly triage multiple alerts ({kibana-pull}181818[#181818]).
+* Creates the **Asset criticality** page within the **Manage** menu, which lets you assign asset criticality levels to your assets ({kibana-pull}179891[#179891]).
+* Adds alert suppression for New Terms rules ({kibana-pull}178294[#178294]).
+* Adds alert suppression for EQL rules with non-sequence queries ({kibana-pull}176422[#176422]).
+* Adds a **Setup guide** markdown field to custom rules ({kibana-pull}178131[#178131]).
+
+[discrete]
+[[enhancements-8.14.0]]
+==== Enhancements
+* Removes the "Technical preview" tag for {esql}; it is now GA ({kibana-pull}180838[#180838]).
+* Allows you to add newly-created fields to an {esql} rule's highlighted fields ({kibana-pull}177746[#177746]).
+* Connects {esql} functionality in {elastic-sec} to the `general:enableESQL` advanced setting ({kibana-pull}181616[#181616]).
+* Makes conversations with Elastic AI Assistant persist across sessions ({kibana-pull}173487[#173487]).
+* Adds conversation streaming for Elastic AI Assistant ({kibana-pull}180095[#180095]).
+* Adds support for Anthropic Claude 3 to the Amazon Bedrock connector, and makes it the default model ({kibana-pull}179304[#179304]).
+* Adds an **AI Assistant** settings section to the **Management** menu ({kibana-pull}176656[#176656]).
+* Adds the `_source` field to the Alert details flyout's JSON view ({kibana-pull}180477[#180477]).
+* Improves the UI for row renderers in Timeline ({kibana-pull}180669[#180669]).
+* Allows data collected by AuditBeat to appear in Session View ({kibana-pull}179985[#179985]).
+* Improves the visual appearance of the asset criticality alert column ({kibana-pull}180868[#180868]).
+* Adds an advanced setting that allows you to turn off alert enrichment from memory scanning for malicious behavior alerts ({kibana-pull}180636[#180636]).
+* Adds an advanced setting that lets you turn off a performance optimization that makes malware on-write and file event processing asynchronous ({kibana-pull}179179[#179179]).
+* Makes the alert details flyout's state persist ({kibana-pull}178746[#178746], {kibana-pull}179511[#179511]).
+* Limits the alerts that can affect an entity's risk score to the 10,000 riskiest ({kibana-pull}178324[#178324]).
+* Adds a tooltip to the **Asset Criticality** section of the Alerts flyout ({kibana-pull}176927[#176927]).
+* Updates prebuilt rule mappings to MITRE ATT&CK version 14.1 ({kibana-pull}174120[#174120]).
```

Comment:
Not sure if this is entirely accurate.

It's not only about prebuilt rules, which indeed the TRaDE team updated to MITRE v14.1, and Kibana v8.14.0 users will get these rules as soon as they _upgrade them_ manually (meaning, it won't happen automatically). Users will be using MITRE v14.1 for custom rules too, when they create new rules or edit existing rules.

Also, to me "Updates ... rule mappings" sounds like we're migrating data of installed/created rules to the new MITRE version, which is not the case. The rules created using prior MITRE versions will stay unchanged.

```suggestion
* Updates MITRE ATT&CK framework to version 14.1 ({kibana-pull}174120[#174120]).
```